components/expenses/ExpensesPage.tsx

```diff
@@ -56,9 +55,9 @@ const Expenses = props => {
     setOldLoggedInUser(LoggedInUser);
   }, [oldLoggedInUser, LoggedInUser, data]);
 
-  const hasFilter = memoizeOne(query => {
```

Comment:
@gustavlrsn Using `memoizeOne` has no effect in a functional component since it will be re-declared at each render.

Reply:
Good to know!